src/Spryker/Glue/AppMerchantBackendApi/Plugin/GlueApplication/MerchantConfirmDisconnectionRequestValidatorPlugin.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Glue\AppMerchantBackendApi\Plugin\GlueApplication;
+
+use Generated\Shared\Transfer\GlueErrorTransfer;
+use Generated\Shared\Transfer\GlueRequestTransfer;
+use Generated\Shared\Transfer\GlueRequestValidationTransfer;
+use Generated\Shared\Transfer\MerchantCriteriaTransfer;
+use Spryker\Glue\AppKernel\Plugin\GlueApplication\AbstractConfirmDisconnectionRequestValidatorPlugin;
+use Spryker\Glue\AppMerchantBackendApi\AppMerchantBackendApiConfig;
+use Spryker\Glue\AppPaymentBackendApi\Mapper\Payment\GlueRequestPaymentMapper;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @method \Spryker\Glue\AppMerchantBackendApi\AppMerchantBackendApiFactory getFactory()
+ */
+class MerchantConfirmDisconnectionRequestValidatorPlugin extends AbstractConfirmDisconnectionRequestValidatorPlugin
+{
+    protected function getLabelOk(): string
+    {
+        return $this->getFactory()->getTranslatorFacade()->trans('Ignore & Disconnect');
+    }
+
+    protected function getLabelCancel(): string
+    {
+        return $this->getFactory()->getTranslatorFacade()->trans('Cancel');
+    }
```

Comment:
I would have those by the Abstract Plugin as default. Only needed to override when it is different which should be the case.

Less code needed to implement and less translations which are duplicated across several packages.

Reply:
Moved out.